src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,84 @@
+"""
+    MaxwellBoltzmann(a)
+
+The *Maxwell-Boltzmann distribution* has probability density function
+```math
+f(x; a) = \\sqrt{\\frac{2}{\\pi}}\\frac{x^2}{a^3}\\exp\\left(-\\frac{x^2}{2a^2}\\right)
+```
+where
+```math
+a = \\sqrt{\\frac{kT}{m}}
+```
+`k` is the Boltzmann constant and `m` is the particle mass.
+
+```julia
+MaxwellBoltzmann()        # Maxwell-Boltzmann distribution with a = 1
+MaxwellBoltzmann(a)       # Maxwell-Boltzmann distribution with parameter a
+MaxwellBoltzmann(T, m)    # Maxwell-Boltzmann distribution with temperature T and mass m
+
+params(d)                 # Get the parameters, i.e. (a,)
+```
+
+External links
+
+* [Maxwell-Boltzmann distribution on Wikipedia](https://en.wikipedia.org/wiki/Maxwell-Boltzmann_distribution)
+"""
+struct MaxwellBoltzmann{T<:Real} <: ContinuousUnivariateDistribution
+    a::T
+    MaxwellBoltzmann{T}(a::T) where {T} = new{T}(a)
+end
+
+function MaxwellBoltzmann(a::T; check_args=true) where {T<:Real}
+    check_args && @check_args(MaxwellBoltzmann, a > zero(a))
+    return MaxwellBoltzmann{T}(a)
+end
+
+MaxwellBoltzmann() = MaxwellBoltzmann(1.0)
+MaxwellBoltzmann(temp::T, mass::T) where {T<:Real} = MaxwellBoltzmann(sqrt(ustrip(k_B) * temp / mass))
```

Comment:
The restriction of temp and mass being of the same type I think is unnecessary. I would leave that without type annotations*. Also, I suggest just explicitly writing the Boltzmann constant in kcal/(mol/K) here to get rid of the additional dependency that was introduced. 

```
const k_Boltzmann = 0.001985875 # kcal mol⁻¹ K⁻¹  

MaxwellBoltzmann() = MaxwellBoltzmann(1.0)
MaxwellBoltzmann(temp, mass) = MaxwellBoltzmann(sqrt(k_Boltzmann * temp / mass))
```


*(a user can try to use an integer for the mass or temperature, for example, and that would not work if the other one is a float).

Reply:
By setting k_B here explicitly one can remove the dependency on PhysicalConstants and the tests pass.

Reply:
Hello!
So, with the suggested edits this implementation would practically work, right? I'd be interested in having a MaxwellBoltzmann distribution added, actually wrote one myself, but this implementation seems more advanced, if it is not stale again. Do you have suggestions what I should do to finally add this to the package (after 5 years and 2 not so successful PRs ^^)? Modify this PR or make a new one? I'm pretty new to working with bigger projects, so any help is appreciated!